module/metrics/hotstuff.go

```diff
@@ -166,24 +196,36 @@ func (hc *HotstuffCollector) HotStuffWaitDuration(duration time.Duration, event
 	hc.waitDuration.WithLabelValues(event).Observe(duration.Seconds()) // unit: seconds; with float64 precision
 }
 
-// HotstuffCollector reports Metrics C8: Current View
+// CountSkipped counts the number of skips we did.
+func (hc *HotstuffCollector) CountSkipped() {
+	hc.skips.Inc()
+}
+
+// CountTimeout counts the number of timeouts we had.
+func (hc *HotstuffCollector) CountTimeout() {
```

Comment:
I find we could improve the clarity of the documentation. From its documentation alone, is unclear, whether this is the number of times the consensus committee left a view with a timeout. It could also mean the number of times this node sent a timeout message. 

I would suggest to update
1. the godoc here 
    ```suggestion
    // CountTimeout tracks the number of views that this replica left due to observing a TC.
    func (hc *HotstuffCollector) CountTimeout() {
    ```
2. Update the metric documentation https://github.com/onflow/flow-go/blob/628b20cd267f34864960ab5834fdaeabddab7900/module/metrics/hotstuff.go#L116 to 
     ```golang
    Help: "The number of views that this replica left due to observing a TC",
    ```